cmd/brew-pull-circle.rb

```diff
@@ -35,7 +35,7 @@ def slug(tap)
     if tap.remote.nil?
       "#{tap.user}/homebrew-#{tap.repo}"
     else
-      x = tap.remote[%r{^https://github\.com/([^.]+)(\.git)?$}, 1]
+      x = tap.remote[%r{^(https://|git@)github\.com(/|:)([^.]+)(\.git)?$}, 3]
```

Comment:
I can’t push to this branch, but I can “suggest” the (hopefully correct) requested change...

```suggestion
      x = tap.default_remote[%r{^(https://|git@)github\.com(/|:)([^.]+)(\.git)?$}, 3]
```